package.json

```diff
@@ -34,37 +14,90 @@
   "keywords": [
     "faces",
     "cartoon",
-    "mii"
+    "mii",
+    "avatar",
+    "svg",
+    "vector",
+    "react",
+    "vite"
   ],
   "homepage": "https://zengm.com/facesjs/",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "test": "jest",
+    "build": "node tools/process-svgs.js && rm -rf build && yarn run build-commonjs && yarn run build-esmodules && yarn run tsc && yarn run tsc -p tsconfig-commonjs.json",
+    "build-commonjs": "cross-env BABEL_ENV=commonjs babel src --extensions '.ts,.js' --out-dir build/commonjs",
+    "build-esmodules": "babel src --extensions '.ts,.js' --out-dir build/esmodules && echo '{\"type\": \"module\"}' > build/esmodules/package.json",
+    "prepare": "yarn run build && husky install",
+    "start": "node tools/server.js"
+  },
+  "jest": {
+    "preset": "ts-jest",
+    "testEnvironment": "node"
+  },
   "devDependencies": {
-    "@babel/cli": "^7.21.0",
-    "@babel/core": "^7.21.0",
-    "@babel/plugin-transform-modules-commonjs": "^7.20.11",
-    "@babel/preset-typescript": "^7.21.0",
-    "@rollup/plugin-babel": "^6.0.3",
-    "@rollup/plugin-node-resolve": "^15.0.1",
-    "@types/node": "^20.8.6",
+    "@babel/cli": "^7.24.1",
+    "@babel/core": "^7.24.4",
+    "@babel/preset-env": "^7.24.4",
+    "@babel/preset-typescript": "^7.24.1",
+    "@nextui-org/react": "^2.2.10",
+    "@phosphor-icons/react": "^2.1.4",
+    "@rewind-ui/core": "^0.19.8",
+    "@tailwindcss/forms": "^0.5.7",
+    "@tailwindcss/typography": "^0.5.12",
+    "@types/jest": "^29.5.12",
+    "@types/node": "^20.12.3",
+    "@types/react": "^18.2.73",
+    "@types/react-dom": "^18.2.23",
+    "@types/react-syntax-highlighter": "^15.5.11",
+    "@vitejs/plugin-react-swc": "^3.6.0",
+    "autoprefixer": "^10.4.19",
     "babel-plugin-add-module-exports": "^1.0.4",
-    "chokidar": "^3.5.3",
+    "canvg": "^4.0.1",
+    "chokidar": "^3.6.0",
+    "class-variance-authority": "^0.7.0",
     "cross-env": "^7.0.3",
-    "husky": "^8.0.3",
-    "lint-staged": ">=10",
-    "prettier": "^3.1.0",
-    "rollup": "^4.5.0",
-    "svgo": "^3.0.2",
-    "typescript": "^5.2.2",
-    "ws": "^8.12.1"
+    "framer-motion": "^11.0.25",
+    "fs": "0.0.1-security",
+    "jest": "^29.7.0",
+    "path": "^0.12.7",
```

Comment:
We don't want this, we want the path built-in. Which you can reference as "node:path" to make it unambiguous when importing, and delete this erroneous dependency.